server/src/packages/pas-form/applicants/applicants.controller.ts

```diff
@@ -15,15 +32,16 @@ export const APPLICANT_ATTRIBUTES = [
   'dcp_state',
   'dcp_zipcode',
   'dcp_phone',
-  'dcp_type',
+  'dcp_type', // only on dcp_applicantinformation entity, not applicantrepresentative
+  'targetEntity', // custom attribute to route to/from correct crm entity
 ];
 
-@UseInterceptors(new JsonApiSerializeInterceptor('applicants', {
-  id: 'dcp_applicantinformationid',
-  attributes: [
-    ...APPLICANT_ATTRIBUTES,
-  ],
-}))
+@UseInterceptors(
+  new JsonApiSerializeInterceptor('applicants', {
```

Comment:
Readability fix from
`prettier --write applicants.controller.ts`

Reply:
I would just find a way to get this working as a linting requirement so it's consistent across.

Now the other controllers will look different.